My project/Assets/Scripts/CreatureLogic/Actions/FindFood.cs

```diff
@@ -84,4 +84,10 @@ public string ToString()
     {
         return "Finding Food";
     }
+
+    public void PrintStatus()
+    {
+        Debug.Log(ToString());
+        Debug.Log("Food Location: " + grid.WorldToCell(food.GetPosition()));
```

Comment:
I know we're currently relying on a ton of debug statements but i think we should move toward having a more standardized way of monitoring the health of the game... i'll let you think about that a bit